zh-CN/mvc/model/object.md

```diff
@@ -14,12 +14,12 @@ o := orm.NewOrm()
 user := new(User)
 user.Name = "slene"
 
-fmt.Println(o.Insert(user))
+fmt.Println(o.Insert(&user))
 
 user.Name = "Your"
-fmt.Println(o.Update(user))
-fmt.Println(o.Read(user))
-fmt.Println(o.Delete(user))
+fmt.Println(o.Update(&user))
+fmt.Println(o.Read(&user))
+fmt.Println(o.Delete(&user))
```

Comment:
wrong fix. it should be user. because user use the new to create. it's already *User.